pkg/storage/storage.go

```diff
@@ -0,0 +1,17 @@
+package storage
+
+import "github.com/dollarshaveclub/psst/pkg/directory"
+
+const (
+	filePrefix = "psst"
+)
+
+// Storage gives us basic methods for storing secrets
+type Storage interface {
```

Comment:
This name also stutters.

Reply:
The names stutter but I don't understand why you'd name the package to something but then need to name the interface something else in this case. I don't know what else to really name them and I'm not sure why stutters are such a problem.

Reply:
Naming isn't easy. I've seen people use `Interface` as the name, I guess when they can't think of anything else. I have mixed feelings about that though, to be honest.